test/e2e/antreapolicy_test.go

```diff
@@ -65,32 +66,13 @@ const (
 	// Verification of deleting/creating resources timed out.
 	timeout = 10 * time.Second
 	// audit log directory on Antrea Agent
-	logDir          = "/var/log/antrea/networkpolicy/"
-	logfileName     = "np.log"
-	defaultTierName = "application"
+	logDir           = "/var/log/antrea/networkpolicy/"
+	logfileName      = "np.log"
+	defaultTierName  = "application"
+	formFactorNormal = "3by3PodWorkloads"
+	formFactorLarge  = "extraNamespaces"
```

Comment:
This is not very easy to understand. After getting how it works, I feel it's perhaps not worth to introduce a "formFactor" parameter to common utils just for sameLabels test. It's also obscure to assume the Namespace labels of the extra Namespaces in TestAntreaPolicyExtendedNamespaces. Following the way, the common setup and teardown code would become coupled with specific tests via more such flags.

Can we create extra namespaces directly in the special tests?